src/stories/page.css

```diff
@@ -0,0 +1,69 @@
+section {
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
Nunito Sans is not our brand font. is this stock-css that comes with storybook? I'd prefer to rely on our defaults from https://github.com/carforyou/carforyou-components-pkg/blob/master/assets/global.css, which is also what we're using in the projects, so default fonts and sizes would be consistent